src/winter_onboarding_2021/fleet_management_service/handlers/user.clj

```diff
@@ -65,6 +77,12 @@
             (wrong-password-response))
           (email-not-found-response))))))
 
+(defn logout [req]
+  (let [session-id (get-in req [:cookies :session-id :value])]
+    (session/delete (java.util.UUID/fromString session-id))
+    (merge (response/redirect "/")
+           {:cookies nil})))
+
```

Comment:
This looks good. Did you make sure that cookies were cleared on client side? Asking because @yogirajh007 & I tried this approach & couldnt get it cleared

Reply:
Yes, I have implemented the same in the updated code.